src/utils/mime.ts

```diff
@@ -70,6 +70,7 @@ const baseMimes: Record<string, string> = {
   woff2: 'font/woff2',
   xhtml: 'application/xhtml+xml',
   xml: 'application/xml',
+  yaml: 'application/yaml',
```

Comment:
Sorry, we have to reconsider it.

if it has another key for YAML mime like `yml`:

```ts
yml: 'application/yaml'
```

If more than one key exists for `application/yaml`, it is not known which extension is used. So, it may be difficult to specify the extension used in SSG by mime type alone.

Reply:
Can I remove this place because it is represented in the DefaultExtentionMap?

Reply:
Yes! You can remove it.